app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -353,6 +353,7 @@ def new_interface(attr = {})
 
     def new_volume(attr = {})
       set_preallocated_attributes!(attr, attr[:preallocate])
+      raise "VM volume attributes are not set properly" unless attr.all? { |key, value| value.is_a? String }
```

Comment:
Should this be a `Foreman::Exception`? Also, maybe indicate how they should be set?

Reply:
@tbrisker 
done. 
If I would write how they should be set it could a bit misleading, In hammer, I'm expecting this all parameters to be received as a big string (in comma-separated values) but in those values, I'm expecting int/string/bool values. 
In the UI it could be a bit different.

Reply:
in that case, would it be possible it fails here when an integer/bool is passed? perhaps this validation should be done in a different place if it differs depending on the source?

Reply:
nope. it shouldn't fail

Reply:
it just that it could get "0" or "false"